jetstream/search-result-de-duplication.toml

```diff
@@ -0,0 +1,95 @@
+[metrics]
+weekly = ["history_impressions", "bookmark_impressions", "search_suggestion_impressions", 
+"history_clicks", "bookmark_clicks", "search_suggestion_clicks", "history_ctr", "bookmarks_ctr", 
```

Comment:
There's a typo here causing the CI check to fail (`bookmarks_ctr` -> `bookmark_ctr`).